### PR TITLE
Translation for PT-BR

### PR DESCRIPTION
**Pull request by CarlinCV**

### CODE REVIEW

File: DestroyObject.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: DestroyObject
+description: Destr�i (remove) um objeto que foi criado usando CreateObject.
```
Comment (Southclaws) [inline DestroyObject.md:3]: looks like unicode errors here
```suggestion
description: Destrói (remove) um objeto que foi criado usando CreateObject.

```

Comment (Southclaws) [inline DestroyObject.md:3]: looks like unicode errors here
```suggestion
description: Destrói (remove) um objeto que foi criado usando CreateObject.

```

File: DestroyObject.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: DestroyObject
+description: Destr�i (remove) um objeto que foi criado usando CreateObject.
+tags: []
+---
+
+## Descri��o
```
Comment (Southclaws) [inline DestroyObject.md:7]:
```suggestion
## Descrição

```

**Review by Southclaws — CHANGES_REQUESTED** [review]
looks good but we need to fix those unicode errors

**Review by iPollo — APPROVED** [review]

**Review by adib-yg — APPROVED** [review]

**Pull request merged**